auto_submit/lib/requests/check_request.dart

```diff
@@ -2,19 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:convert';
-
 import 'package:auto_submit/server/authenticated_request_handler.dart';
 import 'package:auto_submit/service/approver_service.dart';
-import 'package:auto_submit/service/validation_service.dart';
-import 'package:github/github.dart';
 import 'package:googleapis/pubsub/v1.dart' as pub;
 import 'package:shelf/shelf.dart';
 
 import '../request_handling/pubsub.dart';
 
-import '../service/log.dart';
-
 abstract class CheckRequest extends AuthenticatedRequestHandler {
```

Comment:
Is this class still needed as we are separating the two concrete classes into two APIs, and into two apps at the final stage.

Reply:
Technically no but I have kept it around because it houses the pull messages method that is common to both classes. But I can move it to a more appropriate class.

Reply:
Refactored to remove. I just pushed the pubsub call up into the pubsub class.